templates/pocketbase.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Pocketbase</Name>
+  <Repository>lukehagar/pocketbase</Repository>
+  <Registry>https://hub.docker.com/r/lukehagar/pocketbase</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/139061-support-blindkitty38-pocketbase/</Support>
+  <Project>https://pocketbase.io</Project>
+  <Overview>Open Source backend for your next SaaS and Mobile app in 1 file&#xD;
+&#xD;
+Pocketbase docs:&#xD;
+&#xD;
+https://pocketbase.io/docs/&#xD;
+&#xD;
+&#xD;
+SDKs:&#xD;
+&#xD;
+Dart - https://github.com/pocketbase/dart-sdk&#xD;
+JavaScript - https://github.com/pocketbase/js-sdk&#xD;
+&#xD;
+&#xD;
+On the first launch, navigate to the Web UI, which I have configured as the admin interface, and it will prompt you to create the first admin account.&#xD;
+&#xD;
+Admin UI: &#xD;
+http:/{UnraidServerIP}/:{Port}/_/&#xD;
+&#xD;
+API: &#xD;
+http:/{UnraidServerIP}/:{Port}/api/</Overview>
+  <Category>Other: Status:Beta</Category>
+  <WebUI>http://[IP]:[PORT:8080]/_/</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/pocketbase.xml</TemplateURL>
+  <Icon>https://github.com/pocketbase/pocketbase/blob/master/ui/public/images/favicon/mstile-144x144.png</Icon>
+  <Config Name="PUID" Target="PUID" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false">99</Config>
+  <Config Name="PGID" Target="PGID" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false">100</Config>
+  <Config Name="UMASK" Target="UMASK" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false">002</Config>
```

Comment:
I dont see how this is supported in the image

Reply:
Removed